Test/SubscriptionTest.cs

```diff
@@ -305,6 +305,51 @@ public void UpdateSubscription()
             Assert.Equal(newSubscription.NetTermsType, NetTermsType.NET);
         }
 
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void UpdateSubscriptionWithProrationSettings()
+        {
+            var plan = new Plan(GetMockPlanCode(), GetMockPlanName())
+            {
+                Description = "Update Subscription Plan 1"
+            };
+            plan.UnitAmountInCents.Add("USD", 1500);
+            plan.Create();
+            PlansToDeactivateOnDispose.Add(plan);
+
+            var plan2 = new Plan(GetMockPlanCode(), GetMockPlanName())
+            {
+                Description = "Update Subscription Plan 2"
+            };
+            plan2.UnitAmountInCents.Add("USD", 750);
+            plan2.Create();
+            PlansToDeactivateOnDispose.Add(plan2);
+
+            var account = CreateNewAccountWithBillingInfo();
+
+            var sub = new Subscription(account, plan, "USD");
+            sub.Create();
+
+            var subChange = new SubscriptionChange()
+            {
+                PlanCode = plan2.PlanCode,
+                BillingInfoUuid = account.GetBillingInfos()[0].Id,
+                NetTerms = 10,
```

Comment:
**nit:** you can just not set NetTerms at all here, or make it 0 since you're not testing it